src/main/java/de/hysky/skyblocker/utils/render/RenderHelper.java

```diff
@@ -142,6 +142,42 @@ public static void renderOutline(WorldRenderContext context, Box box, float[] co
         }
     }
 
+    /**
+     * Renders the outline of a box with the specified color components and line width.
+     * This does not use renderer since renderer draws outline using debug lines with a fixed width.
+     *
+     * @param alpha the transparency of the lines for the box
+     */
+    public static void renderOutline(WorldRenderContext context, Box box, float[] colorComponents, float alpha, float lineWidth, boolean throughWalls) {
```

Comment:
Instead of duplicating the above method, make the above method have the `alpha` parameter, and add an overload to it that defaults the alpha to 1f